### PR TITLE
I updated the search section to explain that search is an object and …

### PR DESCRIPTION
**Pull request by dino475**
…location is a string with the two options being "side" or "top".

### CODE REVIEW

File: settings/global.mdx [hunk 1]
```diff
@@ -193,8 +193,16 @@ settings. Learn more about the [properties](#properties) or from an
   </Expandable>
 </ResponseField>
 
-<ResponseField name="search" type={'"side" | "top"'}>
-  The location of the search bar entry.
+<ResponseField name="search" type="search">
+  The location options for the search bar.
+  <Expandable title="search">
```
Comment (mayankshouche) [inline settings/global.mdx]: nit: for consistency, can we capitalize the type?
```suggestion
<ResponseField name="search" type="Search">
  The location options for the search bar.
  <Expandable title="Search">
```

**Review by mayankshouche — APPROVED** [review]

**Comment by dino475** [comment]
lets gooooo

**Comment by fmerian** [comment]
awesome! this solves #279!

**Comment by fmerian** [comment]
IMHO that'd be helpful to have a code example here @dino475

**Pull request merged**